projects/rocprofiler-sdk/source/include/rocprofiler-sdk/agent.h

```diff
@@ -210,6 +241,8 @@ typedef struct rocprofiler_agent_v0_t
     int32_t                               logical_node_type_id;
     rocprofiler_agent_runtime_visiblity_t runtime_visibility;
     rocprofiler_uuid_t                    uuid;  ///< GPU only. Universally unique identifier.
+    rocprofiler_agent_fw_info_t firmware_info;  ///< GPU only. Detailed firmware version information
```

Comment:
```suggestion
    rocprofiler_agent_firmware_info_v0_t fw_info;  ///< GPU only. Detailed firmware version information
```
In the future, if we need to add more firmware info, we will create `rocprofiler_agent_fw_info_v1_t` and add a field `rocprofiler_agent_fw_info_v1_t firmware_info_v1;`. Also, sure `firmware` in struct name and abbreviation (`fw`) in variable name.